app/code/Magento/Ui/view/base/web/js/dynamic-rows/dynamic-rows.js

```diff
@@ -548,6 +548,13 @@ define([
                     });
```

Comment:
It's would be nice if you will mark [dynamic-rows-tier-price.js](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Catalog/view/adminhtml/web/js/components/dynamic-rows-tier-price.js#L6-L29) as deprecated, because it was created by mistake for fix same issue, but just for tier price functionality.